test/srt/layers/attention/mamba/test_causal_conv1d.py

```diff
@@ -13,6 +13,10 @@
     causal_conv1d_fn,
     causal_conv1d_update,
 )
+from sglang.test.test_utils import empty_gpu_cache
+
+device_type = getattr(torch.accelerator.current_accelerator(), "type", "cpu")
+torch.set_default_device(device_type)
```

Comment:
I feel like you could add a util function for this device setting in `python/sglang/test/test_utils.py` and then you can reuse this util function in all these files.